src/test/java/com/pontusvision/gdpr/PVVisJSGraphTests.java

```diff
@@ -0,0 +1,132 @@
+//                    "group":"Object.Module",
+//                    "label":"lyyyy",
+//                    "shape":"image",
+//                    "image":
+//              "eyJNZXRhZGF0YS5UeXBlIjpbIk9iamVjdC5Nb2R1bGUiXSwiTWV0YWRhdGEuVHlwZS5PYmplY3QuTW9kdWxlIjpbIk9iamVjdC5Nb2R1bGUiXSwiT2JqZWN0Lk1vZHVsZS5OYW1lIjpbImx5eXl5Il19"
+//            },
+//            {
+//              "id":"#142:0",
+//                    "group":"Object.Data_Source",
+//                    "label":"SISTEMA DE CRM PARA CAPTURA DE LYYYY",
+//                    "shape":"image",
+//                    "image":
+//              "eyJPYmplY3QuRGF0YV9Tb3VyY2UuVHlwZSI6WyJTYWFTIl0sIk9iamVjdC5EYXRhX1NvdXJjZS5VUklfUmVhZCI6WyJodHRwczovL2FwcC54eHh4eC5jb20vIl0sIk9iamVjdC5EYXRhX1NvdXJjZS5EZXNjcmlwdGlvbiI6WyJTSVNURU1BIERFIENSTSBQQVJBIENBUFRVUkEgREUgTFlZWVkiXSwiT2JqZWN0LkRhdGFfU291cmNlLkZvcm1fSWQiOlsiMSJdLCJPYmplY3QuRGF0YV9Tb3VyY2UuVVJJX1VwZGF0ZSI6WyJodHRwczovL2FwcC54eHh4eC5jb20vIl0sIk1ldGFkYXRhLlR5cGUuT2JqZWN0LkRhdGFfU291cmNlIjpbIk9iamVjdC5EYXRhX1NvdXJjZSJdLCJNZXRhZGF0YS5UeXBlIjpbIk9iamVjdC5EYXRhX1NvdXJjZSJdLCJPYmplY3QuRGF0YV9Tb3VyY2UuVVJJX0RlbGV0ZSI6WyJodHRwczovL2FwcC54eHh4eC5jb20vIl0sIk9iamVjdC5EYXRhX1NvdXJjZS5OYW1lIjpbIlNJU1RFTUEgREUgQ1JNIFBBUkEgQ0FQVFVSQSBERSBMWVlZWSJdLCJPYmplY3QuRGF0YV9Tb3VyY2UuRG9tYWluIjpbIkx5eXl5Il0sIk9iamVjdC5EYXRhX1NvdXJjZS5FbmdpbmUiOlsieHh4eHgiXX0="
+//            },
+//            {
+//              "id":"#238:0",
+//                    "group":"Object.System",
+//                    "label":"XXXXX",
+//                    "shape":"image",
+//                    "image":
+//              "eyJNZXRhZGF0YS5UeXBlIjpbIk9iamVjdC5TeXN0ZW0iXSwiT2JqZWN0LlN5c3RlbS5OYW1lIjpbIlhYWFhYIl0sIk1ldGFkYXRhLlR5cGUuT2JqZWN0LlN5c3RlbSI6WyJPYmplY3QuU3lzdGVtIl19"
+//            },
+//            {
+//              "id":"#234:0",
+//                    "group":"Object.Subsystem",
+//                    "label":"SAAS",
+//                    "shape":"image",
+//                    "image":
+//              "eyJNZXRhZGF0YS5UeXBlIjpbIk9iamVjdC5TdWJzeXN0ZW0iXSwiTWV0YWRhdGEuVHlwZS5PYmplY3QuU3Vic3lzdGVtIjpbIk9iamVjdC5TdWJzeXN0ZW0iXSwiT2JqZWN0LlN1YnN5c3RlbS5OYW1lIjpbIlNBQVMiXX0="
+//            }
+//            ],
+//          "edges": [
+//            {
+//              "from":"#234:0",
+//                    "to":"#190:0",
+//                    "label":"Has Subsystem"
+//            },
+//            {
+//              "from":"#190:0",
+//                    "to":"#142:0",
+//                    "label":"Has Module"
+//            },
+//            {
+//              "from":"#238:0",
+//                    "to":"#234:0",
+//                    "label":"Has System"
+//            }
+//            ]
+//        }
+
+      JSONObject obj = new JSONObject(infraGraphJson);
+      String resp = "| ";
+      JSONArray array = obj.getJSONArray("nodes");
+
+      for(int i = 0 ; i < array.length() ; i++){
+        resp += array.getJSONObject(i).getString("group");
```

Comment:
good start, but needs more work; consider that the graph order may change each time, so you may want  to create a list and sort it.  Also, we need to check the edges as well.
Consider doing this:

1. create a Map <string,string> with he id as the key, and the group as the value
2. traverse all the edges, and replace the to/from id with the value from the Map in (1)
3. create a list of strings similar to this: <from group> --- <edgelabel> --- <to group>
4. order the list so the test can pass even when the JSON order changes.